### DISCUSSION TITLE
About the Tag Component

### DISCUSSION THREAD

**Discussion by xiatian12345** [carbon-design-system/carbon-components-svelte#1953] [Q&A]
When add close event,the click event is auto disabled,how should i do to retain the click and close event?

**Comment by metonym** [comment]
Fixed in [v0.102.0](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.102.0)

For a filterable `Tag`, clicking the body forwards `click` and `mouse` events, while clicking the close button will only fire the "close" event.

https://svelte.carbondesignsystem.com/components/Tag#filterable-with-click-and-close
